packages/labs/test/protobufjs/test.ts

```diff
@@ -0,0 +1,11 @@
+import {TestMessage} from 'build_bazel_rules_nodejs/packages/labs/test/protobufjs/test_ts_proto';
+
+// const TestMessage = build_bazel_rules_nodejs.packages.labs.test.protobufjs.TestMessage;
+
+describe('protobufjs', () => {
+  it('should work in node', () => {
+    expect(TestMessage.verify({
+      testField: 'Hello',
+    })).toBeFalsy();  // verify returns an error if failed
```

Comment:
should there be a success case too? or is it enough to assert that it doesn't throw an error?

Reply:
This test case is really just to run some code - I could change it to `expect(TestMessage).toBeTruthy();` and get the verification needed for this PR, would you prefer the simpler test case?